src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -36,6 +39,9 @@
     nova_client_exists = False
 
 
+DEFAULT_OPENSTACK_PROVIDER_NAME = 'openstack'
```

Comment:
Yes and no :) As you can see [here](https://github.com/allegro/ralph/pull/2867/files#diff-4fa30bd6416c1bceb147b0bf9dbf6999L237), it was harcoded value previously.